back/src/Guide/Database/Get.hs

```diff
@@ -217,6 +247,18 @@ getCategoryMaybe catId = do
         pure $ Category{..}
   lift $ HT.statement catId (Statement sql encoder decoder False)
 
+-- | Get 'items_order' of category.
+getCategoryItemsOrder :: Uid Category -> ExceptT DatabaseError Transaction [Uid Item]
+getCategoryItemsOrder catId = do
+  let sql = [r|
+        SELECT items_order
+        FROM categories
+        WHERE uid = $1
+        |]
+      encoder = uidParam
+      decoder = HD.singleRow uidsColumn
```

Comment:
`singleRow` can fail. Use `rowMaybe` and throw `CategoryNotFound` when necessary.